src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -109,14 +91,8 @@ export const VenueLandingPage: React.FC = () => {
     return <LoadingPage />;
   }
 
-  const isUserVenueOwner = user && venue.owners?.includes(user.uid);
-
   const nextVenueEventId = futureOrOngoingVenueEvents?.[0]?.id;
```

Comment:
What is `nextVenueEventId` needed/used for still? Can it be removed/cleaned up here? Or is it still needed?

Reply:
@mike-lvov Can you help me regarding this question?

Reply:
@0xdevalias legacy logic related to 
![image](https://user-images.githubusercontent.com/63194656/125629515-559c8d83-be9f-4246-8136-a4abbb219c96.png)